test/functional/rpc_net.py

```diff
@@ -249,11 +249,17 @@ def test_addnode_getaddednodeinfo(self):
         added_nodes = self.nodes[0].getaddednodeinfo()
         assert_equal(len(added_nodes), 1)
         assert_equal(added_nodes[0]['addednode'], ip_port)
+        # check that filtering by node works
+        self.nodes[0].addnode(node="11.22.33.44", command='add')
+        added_node = self.nodes[0].getaddednodeinfo(node=ip_port)
```

Comment:
nit:  If this file is touched, would recommend using `first_added_node` or `added_node_0` instead of `added_node`  to make it clearer that the node being referred to is the one added before 11.22.33.44.  Alternatively, could update the comment/log message instead.

Reply:
`added_node_0` seems weird because it can be confused with nodes[0], I prefer `first_added_node`. I will address it.